src/devtools/client/debugger/src/components/Editor/index.tsx

```diff
@@ -464,40 +520,9 @@ class Editor extends PureComponent {
   }
 }
 
+// @ts-expect-error context shenanigans
```

Comment:
Makes me a little sad. Since we never use context I wonder if it would be reasonable to make Context's type `CurrentContextType | {}` and just start passing `{}` everywhere until we actually pull it out. Probably a bad idea.

Reply:
The issue here is TS arguing "you defined a class above here, but you didn't say it would have a `.contextTypes` field attached!"

(the real solution is "DON'T USE LEGACY CONTEXT!" :)  )